.travis.yml

```diff
@@ -6,7 +6,7 @@ scala:
 jdk:
     - oraclejdk8
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION coverage test tut
+  - sbt ++$TRAVIS_SCALA_VERSION coverage test tut doc
```

Comment:
I think there's no problem in compiling docs for all versions. Scaladoc features may not change that much, but I guess we still want to ensure our docs are compatible with all Scala versions.

That said, we can generate docs for 2.12.x only just as we check the tut output:

```
  - if [[ $TRAVIS_SCALA_VERSION =~ '^2\.12.*' ]]; then
      sbt ++$TRAVIS_SCALA_VERSION doc;
      fi
```